docs/user-guide/projects/project-settings/versions/README.md

```diff
@@ -18,7 +18,7 @@ Also, version will  be used to enable the Backlogs module, i.e. to create a prod
 
 Press the green **+ Version** button to create a new version for a project.
 
-![User-guide-project-settins-versions](User-guide-project-settins-versions.png)
+![User-guide-project-settings-versions](User-guide-project-settins-versions.png)
```

Comment:
I did not change the file name. There seem to be 2 files:
```
User-guide-project-settings-versions.png
User-guide-project-settins-versions.png
```

The one with the typo seems to be later. It has Version 1.2 in the example.

I can make a separate PR to sort that out.

Reply:
Done